### ISSUE TITLE
custom meat user

### ISSUE THREAD

**Issue by foxpcteam** [sabbir1991/WP-User-Quick-Edit#3]
Hi,
great project.

what about custom meat user?

**Comment by sabbir1991** [comment]
Yes, You can, just use the hook ```wp_user_quick_edit_custom_fields``` and extends the ```UserInlineEdit``` js object. Similarly as WP Post

**Comment by sabbir1991** [comment]
Yes, You can, just use the hook `wp_user_quick_edit_custom_fields` and extends the `UserInlineEdit` js object. Similarly as WP Post

**Comment by robertandrews** [comment]
I wonder if you could provide a more detailed description, or link?
Many thanks, this looks great.
